docs/docs.less

```diff
@@ -14,6 +14,12 @@
 */
 @import (reference) 'vars';
 
+#content {
+  &.homepage {
```

Comment:
17 and 18 can be combined and simplified as `#homepage { ... }`

Reply:
class needed